### PR TITLE
[BUG] Bind To Issues

### PR DESCRIPTION
**Pull request by dtiwarATS**

### CODE REVIEW

File: assets/src/components/parts/janus-capi-iframe/ExternalActivity.tsx [hunk 1]
```diff
@@ -13,25 +13,15 @@ import { PartComponentProps } from '../types/parts';
 import { getJanusCAPIRequestTypeString, JanusCAPIRequestTypes } from './JanusCAPIRequestTypes';
 import { CapiIframeModel } from './schema';
 
-const externalActivityMap: Map<string, any> = new Map();
 let context = 'VIEWER';
-const getExternalActivityMap = () => {
```
Comment (dtiwarATS) [inline assets/src/components/parts/janus-capi-iframe/ExternalActivity.tsx:18]: We don't need this anymore.

File: assets/src/components/parts/janus-capi-iframe/ExternalActivity.tsx [hunk 2]
```diff
@@ -919,9 +918,19 @@ const ExternalActivity: React.FC<PartComponentProps<CapiIframeModel>> = (props)
     if (!simLife.init) {
       sendFormedResponse(simLife.handshake, {}, JanusCAPIRequestTypes.INITIAL_SETUP_COMPLETE, {});
       simLife.init = true;
+      if (screenContext === NotificationType.CONTEXT_CHANGED) {
```
Comment (dtiwarATS) [inline assets/src/components/parts/janus-capi-iframe/ExternalActivity.tsx:921]: for bind To variables, the SIM expects a value change request after we have completed the initial setup. This value is considered as the base value by that SIM. When a new values is received, that gets displayed in the textbox

File: assets/test/adaptivity/scripting_test.ts [hunk 3]
```diff
@@ -247,6 +248,31 @@ describe('Scripting Interface', () => {
       expect(valuey).toBe(varValueFormat2);
     });
 
+    it('should return the CSS as it is', () => {
```
Comment (dtiwarATS) [inline assets/test/adaptivity/scripting_test.ts:251]: This is from an old PR that was already approved by @bsparks . By Mistake, i made these changes in Master branch and had created the PR on Master so closed that one and added this here.

**Comment by bsparks** [comment]
can't we just *not* send the "bind to" as init state facts at all? I feel nervous about adding this special logic, I don't think there is anything specifically about bind to in the original code. makes me think that something is off with the flow already

**Comment by dtiwarATS** [comment]
Hey @bsparks , I have made some changes in the existing PR to remove few extra lines of code. As far as the 'bind to' logic in External Activity, I am not sure what else we can do. We can talk about it on the call

**Comment by dtiwarATS** [comment]
Closing this PR. I have created another one

**Pull request closed without merging**